pager-indicators/src/main/java/com/google/accompanist/pager/PagerIndicator.kt

```diff
@@ -162,6 +269,103 @@ fun VerticalPagerIndicator(
     spacing: Dp = indicatorHeight,
     indicatorShape: Shape = CircleShape,
 ) {
+    val stateBridge = remember(pagerState) {
+        object : PagerStateBridge {
+            override val currentPage: Int
+                get() = pagerState.currentPage
+            override val currentPageOffset: Float
+                get() = pagerState.currentPageOffset
+        }
+    }
+
+    VerticalPagerIndicator(
+        pagerState = stateBridge,
+        pageCount = pageCount,
+        modifier = modifier,
+        pageIndexMapping = pageIndexMapping,
+        activeColor = activeColor,
+        inactiveColor = inactiveColor,
+        indicatorHeight = indicatorHeight,
+        indicatorWidth = indicatorWidth,
+        spacing = spacing,
+        indicatorShape = indicatorShape
+    )
+}
+
+/**
+ * An vertically laid out indicator for a [VerticalPager] or [HorizontalPager], representing
+ * the currently active page and total pages drawn using a [Shape].
+ *
+ * This element allows the setting of the [indicatorShape], which defines how the
+ * indicator is visually represented.
+ *
+ * @param pagerState The Compose Foundation PagerState object of your [Pager] to be used to observe
```

Comment:
maybe instead of "The Compose Foundation PagerState" we will have a [androidx.compose.foundation.pager.PagerState]